content/assets/javascripts/application.js

```diff
@@ -1 +1,21 @@
 //= require bootstrap-sprockets
+
+jQuery(document).ready(function($){
+    var back_to_top = $('.scroll-top');
+
+    $(window).scroll(function(){
+        if ($(this).scrollTop() > 100 )
+            back_to_top.addClass('scroll-is-visible')
```

Comment:
best to use `bootstrap` classes for this. You can add an other class for animation purpose, best to use https://daneden.github.io/animate.css